recipe/meta.yaml

```diff
@@ -7,18 +7,18 @@ package:
 source:
   fn: webcolors-{{ version }}.tar.gz
   url: https://pypi.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
-  sha256: 08b07af286a01bcd30d583a7acadf629583d1f79bfef27dd2c2c5c263817277d
+  sha256: ecb3d768f32202af770477b8b65f318fa4f566c22948673a977b00d589dd80f6
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python {{ python_min }}
     - pip
-    - setuptools
+    - pdm-backend
```

Comment:
* Upstream changed from `setuptools` to  [`pdm-backend`]( https://github.com/ubernostrum/webcolors/blob/24.11.1/pyproject.toml#L2 )
* Grayskull caught this when run locally
* Somehow the update bot using Grayskull did not